src/map/status.c

```diff
@@ -3163,11 +3199,16 @@ int status_calc_mercenary_(struct mercenary_data *md, enum e_status_calc_opt opt
 	return 0;
 }
 
-int status_calc_elemental_(struct elemental_data *ed, enum e_status_calc_opt opt) {
-	struct status_data *estatus = &ed->base_status;
-	struct s_elemental *ele = &ed->elemental;
-	struct map_session_data *sd = ed->master;
+int status_calc_elemental_(struct elemental_data *ed, enum e_status_calc_opt opt)
+{
+	struct status_data *estatus;
+	struct s_elemental *ele;
+	struct map_session_data *sd;
 
+	nullpo_ret(ed);
+	estatus = &ed->base_status;
+	ele = &ed->elemental;
+	sd = ed->master;
 	if ( !sd )
```

Comment:
Can you please change this to sd == NULL (i know it's not included in the PR but would be nice addition)